index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length ++
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items[pos]===undefined){
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos]
+  }
 
-  max() {}
+  max() {
+    if(this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
 
-  min() {}
+    this.items.sort(function(a, b) {
+      return b - a;
+    });
+    return this.items[0]
+  }
 
-  sum() {}
+  min() {
+    if(this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
 
-  avg() {}
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+    return this.items[0]
+  }
+
+  sum() {
+    if(this.items.length === 0){
+      return 0
+    }
+    return this.items.reduce((a, b)=> a+b)
+  }
+
+  avg() {
+    if(this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
+    let total = this.items.reduce((a, b)=> a+b)
```

Comment:
Once you have the **sum()** already above, you could just divide it by the length 👍